libs/community/langchain_community/vectorstores/milvus.py

```diff
@@ -381,6 +383,7 @@ def _create_collection(
                 schema=schema,
                 consistency_level=self.consistency_level,
                 using=self.alias,
+                shards_num=self.shards_num,
```

Comment:
`shards_num` looks like it is [deprecated](https://github.com/milvus-io/pymilvus/blob/b6906e1790f2f1e7499a50831fcd456af34c0586/pymilvus/orm/collection.py#L81) in Milvus in favor of `num_shards` -- is that right? Does that parameter work equivalently for you?

Reply:
Yes, correct, i was using old version of pymilvus